dotnet-engine/Yggdrasil.Engine.Tests/YggdrasilEngineTest.cs

```diff
@@ -14,10 +15,10 @@ public class Tests
         PropertyNamingPolicy = JsonNamingPolicy.CamelCase
     };
 
-    [Test]
+    //[Test]

```

Comment:
What's the plan here?

If we want to keep this, let's leave it in. If it needs refactor to make this easier and saner to use, let's do that in another PR

```suggestion
    [Test]

```

Reply:
The plan is to ensure that our builds run quickly while we look at a good way to separate tests to be run at build and tests to be run when explicitly invoked, and that seems to be test categories but I wanted to save that for a new PR when I've had a chance to try it out properly

Reply:
Can we remove the commented out stuff then?

Not gonna block this for that though